app/uk/gov/hmrc/fhregistrationfrontend/views/businessPartners/v2/business_partner_enter_address.scala.html

```diff
@@ -0,0 +1,166 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
+@import uk.gov.hmrc.fhregistrationfrontend.forms.models.Address
+@import uk.gov.hmrc.fhregistrationfrontend.views.helpers._
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.helpers._
+@import uk.gov.hmrc.fhregistrationfrontend.forms.definitions.BusinessPartnersEnterAddressForm._
+@import uk.gov.hmrc.fhregistrationfrontend.views.ViewHelpers
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(viewHelpers: ViewHelpers,
+layout: layout,
+formHelper: FormWithCSRF,
+textInput: components.text_field
+)
+
+@(form: Form[Address],postAction: Call, partnerName: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+
+@AddressContent = @{AddressUK(
+form,
+AddressUKParams(
+name = s"form.$enterAddressKey",
+label = Messages("fh.business.partners.enterAddress.title","company"),
+hasErrors = form(s"form.postcode").hasErrors,
+error = Some(Messages("fh.business_partners.enter.partner_address.error"))
+),
+viewHelpers = viewHelpers
+)}
+
+
+@fieldName = @{s"form.$enterAddressKey"+"-title"}
+
+
+@layout(title = messages("fh.business.partners.enterAddress.title"), backUrl = Some("#")){
+
+
+@viewHelpers.form(action = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.BusinessPartnersChooseAddressController.next, 'novalidate -> "novalidate"){
+@if(form.errors.nonEmpty) {
+@viewHelpers.govukErrorSummary(
+ErrorSummary(errorList = Seq(ErrorLink(
+href = Some(s"#${form.errors.head.key}"),
+content = Text(messages(s"fh.${form.errors.head.key}.${form.errors.head.message}"))
+))
+,
+title = Text(messages("fh.generic.error"))))
+}
+
+
+@formHelper(action = postAction){
+
+<fieldset class="govuk-fieldset" xmlns="http://www.w3.org/1999/html">
+    <legend id="fieldName" class="govuk-fieldset__legend govuk-fieldset__legend--m">
+        <h1 class="govuk-fieldset__heading">
+            <span class="govuk-caption-l business-partner-individual-address-caption" id="business-partner-address">@Messages("fh.businessPartners.caption")</span>
+            <span class="govuk-heading-l govuk-!-margin-bottom-0" id="page-heading">@Messages("fh.business.partners.enterAddress.h1",partnerName)</span>
+        </h1>
+    </legend>
+    </br>
+    @viewHelpers.govukInput(Input(
+    id = s"$fieldName.Line1",
+    name = s"$fieldName.Line1",
+    label = Label(
+    isPageHeading = false,
+    classes = "govuk-!-font-weight-bold",
```

Comment:
It might be worth confirming with the design guys but looking at the screenshot on the ticket I don't _think_ these labels should be bold.